src/features/domain-preview/DomainPreview.tsx

```diff
@@ -1,126 +1,219 @@
+		isLoading: isLoadingMetadata,
+	};
+
+	const altTemplate = `${metadata?.title ?? zna} nft `;
+
+	const thumbnailAlt = altTemplate + ` thumbnail`;
+	const thumbnailSrc = metadata?.previewImage ?? metadata?.image;
+
+	const bannerAlt = altTemplate + ` banner`;
+	const bannerSrc = metadata?.image_full ?? metadata?.image;
+
+	/***************************
+	 * Render
+	 ***************************/
+
 	return (
 		<div className={styles.Container}>
-			{/* TODO: media asset component */}
-
 			<div
 				className={cx(styles.Banner, {
-					isNFTView: isNFTView,
+					isNFTView: variant === 'full',
 				})}
 			>
 				<IpfsMedia
-					alt={`${title ?? 'loading'} nft image banner`}
+					alt={bannerAlt}
 					className={styles.Media}
-					src={banner}
-					options={{
-						size: isNFTView ? 'full' : 'large',
-						fit: isNFTView ? 'fit' : 'fill',
-					}}
+					src={bannerSrc}
+					options={BannerOptions[variant]}
 				/>
 			</div>
 
 			<div
 				className={cx(styles.Content, {
-					isNFTView: isNFTView,
+					isNFTView: variant === 'full',
 				})}
 			>
-				{/* TODO: media asset component */}
-				{!isNFTView && (
-					<IpfsMedia
-						alt={`${title ?? 'loading'} nft thumbnail`}
-						className={styles.Icon}
-						src={icon}
-						options={{
-							size: 'medium',
-						}}
-					/>
+				{variant === 'minimal' && (
+					<Thumbnail alt={thumbnailAlt} src={thumbnailSrc} />
 				)}
+
 				<div className={styles.TextContainer}>
-					<SkeletonText
-						as={'h1'}
-						className={styles.Title}
-						asyncText={{ text: title, isLoading: !title }}
-						skeletonOptions={{ width: '50%' }}
-					/>
-					{/*{title && <h1 className={styles.Title}>{title}</h1>}*/}
-
-					{/* TODO: member component */}
-					{isNFTView && (
-						<ul className={styles.MemberContainer}>
-							{members.map((member) => (
-								<li key={member.title} className={styles.MemberItem}>
-									<span className={styles.MemberTitle}>{member.title}</span>
-									<SkeletonText
-										as={'span'}
-										className={styles.MemberAddress}
-										asyncText={{
-											text: member?.address
-												? truncateAddress(member.address)
-												: undefined,
-											isLoading: !member?.address,
-										}}
-									/>
-								</li>
-							))}
-						</ul>
-					)}
+					<Title title={title} />
 
-					<SkeletonText
-						as={'p'}
-						asyncText={{ text: description, isLoading: !description }}
-						className={styles.Description}
-					/>
+					{variant === 'full' && <Members members={members} />}
 
-					{href && (
-						<div className={styles.LinkContainer}>
-							{/* TODO: arrow link component */}
-							<Link className={styles.Link} to={href}>
-								{'View Domain NFT ->'}
-							</Link>
-						</div>
+					<Description description={description} />
+
+					{variant === 'minimal' && (
+						<NftViewLink to={`/${zna}/nfts?view=true`} />
 					)}
 				</div>
 			</div>
 		</div>
 	);
 };
+
+/*******************
```

Comment:
Yep like this format of the file too!

Reply:
Agreed - file is a bit longer but it keeps scope accurate.